inference-engine/src/gna_plugin/optimizer/gna_pass_manager.cpp

```diff
@@ -782,8 +778,9 @@ void InsertConcatAligningFilterPass::run() {
                 size_t num_rows_out = num_rows_padded + num_rows_in;
 
                 // encodes offset to beginning of split layer input
+                size_t off = (aligned64_offset / bytesPerConcatElement) * (quantized ? bytesPerConcatElement : 4);
```

Comment:
rename off to something useful like bytesOffset

Reply:
ok